scripts/post.sh

```diff
@@ -43,10 +43,16 @@ fi
 # Lets brand this a bit
 
 cat <<'EOT' >> /home/vagrant/.bashrc
-
+echo '     __                                           '
+echo '  | /  \ |      Welcome to the         //  \\   '
+echo ' \_\\  //_/   Islandora 7.x-1.12      _\\()//_  '
+echo '   //()\\     Release Candidate 1    / //  \\ \ '
+echo '   \\  //                             | \__/ |  '
+echo '              "Roll initiative!"                '
+echo '                                                '
```

Comment:
Beautiful

Reply:
@dannylamb Just trying to keep up the tradition.